docs/source/guide/tasks.md

```diff
@@ -4,7 +4,7 @@ short: Import data
 type: guide
 tier: all
 order: 120
-order_enterprise: 105
+order_enterprise: 
```

Comment:
is it ok to have this empty?

Reply:
@makseq https://github.com/heartexlabs/label-studio/pull/3442/files#diff-71883e6254217158bd24bf5a5629ff8eff44d0b4c54d8600a74216442200e32cR7